### PR TITLE
Kathirsvn/api version update

### PR DESCRIPTION
**Pull request by kathirsvn**
**What this PR does**:
Updated JSON API and the coordinator version to the latest and fixed the relevant test cases.

**Checklist**
- [X] Changes manually tested
- [X] Automated Tests added/updated
- [X] Documentation added/updated
- [X] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: tests/collections/collection.test.ts [hunk 1]
```diff
@@ -116,7 +115,7 @@ for (const testClient in testClients) {
           error = e;          
         }
         assert.ok(error);
-        assert.strictEqual(error.errors[0].message, "insertMany can not take more than 100 docs");
+        assert.strictEqual(error.errors[0].message, "Request invalid, the field postCommand.command.documents not valid: amount of documents to insert is over the max limit.");
```
Comment (tatu-at-datastax) [inline tests/collections/collection.test.ts:118]: This is probably not on client side, but error messages should ideally contain information on what the limit is, as well as how many were sent. I'll file an issue on json-api to see if we could improve this

Comment (tatu-at-datastax) [reply tests/collections/collection.test.ts:118]: Filed https://github.com/stargate/jsonapi/issues/365 and will submit a PR adding missing information

**Review by tatu-at-datastax — COMMENTED** [review]
LGTM.

**Review by tatu-at-datastax — APPROVED** [review]

File: tests/collections/collection.test.ts (hunk 1 above)
Comment (kathirsvn) [reply tests/collections/collection.test.ts:118]: Thanks @tatu-at-datastax The new message from the JSON API will reflect in the stargate-mongoose side once the next version of JSON API is published and that gets updated here as well.

**Pull request merged**